### ISSUE TITLE
PedestrianDetector.ProcessAndRender function does not create output parameter.

### ISSUE THREAD

**Issue by gferrell** [emgucv/emgucv#573]
The PedestrianDetector.ProcessAndRender method  empty output parameter.  The result is always null because the code is commented out in this source: https://github.com/emgucv/emgucv/blob/master/Emgu.CV.Models/PedestrianDetector.cs

**Comment by gferrell** [comment]
Can someone note this bug and let me know how to get a fixed version?

**Comment by emgucv** [comment]
It is not a bug and is by design. We have updated the documentation in the following commit to explain what the function will do to the output image:

https://github.com/emgucv/emgucv/commit/485d7e82de03d9ee2c41d9aafb3870f552a5d3c4

For PedestrianDetector, the new documentation says:

> The output image, can be the same as "imageIn", in which case we will render directly into the input image. Note that if no pedestrian is detected, "imageOut" will remain unchanged. If a pedestrian is detected, we will draw the (rectangle) region on top of the existing pixels of "imageOut".  If the "imageOut" is not the same object as "imageIn", it is a good idea to copy the pixels over from the input image before passing it to this function.

If the "imageOut" is blank and no person has been detected it will remain a blank image. If a person is detected the retangle region will be drawn on top of the "imageOut" image.

**Comment by emgucv** [comment]
Futher more, if you download the release package (v4.5.3) in ".zip" or ".exe" file here:
https://github.com/emgucv/emgucv/releases
Open the visual studio solution under "Solution\Windows.Desktop\Emgu.CV.Example.sln", compile and run the "XamarinForms.WPF" project, the click the "pedestrian detection" button and you will see the code in action:
![image](https://user-images.githubusercontent.com/2035816/131746235-21682709-5d31-48fa-afd4-1d2047fa2a50.png)

FYI, the pedestrian detection demo use HOG to detect pedestrian. You may want to check out the Yolo (DNN module) demo, which can be faster and more accurate in detecting pedestrians.

**Comment by gferrell** [comment]
> 
> 
> Futher more, if you download the release package (v4.5.3) in ".zip" or ".exe" file here:
> https://github.com/emgucv/emgucv/releases
> Open the visual studio solution under "Solution\Windows.Desktop\Emgu.CV.Example.sln", compile and run the "XamarinForms.WPF" project, the click the "pedestrian detection" button and you will see the code in action:
> ![image](https://user-images.githubusercontent.com/2035816/131746235-21682709-5d31-48fa-afd4-1d2047fa2a50.png)
> 
> FYI, the pedestrian detection demo use HOG to detect pedestrian. You may want to check out the Yolo (DNN module) demo, which can be faster and more accurate in detecting pedestrians.

I would like to try out the the best pedestrian detection this tool can provide. Is the pedestrian detection shown in Solution\Windows.Desktop\Emgu.CV.Example.sln the YOLO demo? I can't get it to compile.  I get error:
NETSDK1004	Assets file 'C:\Users\geoff\Downloads\2\Emgu.CV.Example\XamarinForms\WPF\obj\project.assets.json' not found. Run a NuGet package restore to generate this file.	XamarinForms.WPF	C:\Program Files\dotnet\sdk\5.0.400\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets	241

**Comment by emgucv** [comment]
It is exactly like the error message says. You need to run a nuget restore on the solution first.

The XamarinForms.WPF project has both HOG detection and YOLO.

**Comment by gferrell** [comment]
That is very interesting that it has HOG and YOLO.  Is there a writeup about the demo?

I decided to do a fresh download from https://github.com/emgucv/emgucv/releases and try again.  I got a couple of errors.  The projects referenced are present, so I'm not sure why it's complaining.:

NU1105	Unable to find project information for 'C:\Users\geoff\source\repos\EMGU-CUDA\Emgu.CV.Models\Emgu.CV.Models.NetStandard\Emgu.CV.Models.NetStandard.csproj'. If you are using Visual Studio, this may be because the project is unloaded or not part of the current solution so run a restore from the command-line. Otherwise, the project file may be invalid or missing targets required for restore.	XamarinForms.WPF	C:\Users\geoff\source\repos\EMGU-CUDA\Emgu.CV.Example\XamarinForms\WPF\XamarinForms.WPF.csproj	1	

Error	NU1105	Unable to find project information for 'C:\Users\geoff\source\repos\EMGU-CUDA\Emgu.CV.Platform\NetStandard\Emgu.CV.Platform.NetStandard.csproj'. If you are using Visual Studio, this may be because the project is unloaded or not part of the current solution so run a restore from the command-line. Otherwise, the project file may be invalid or missing targets required for restore.	XamarinForms.WPF	C:\Users\geoff\source\repos\EMGU-CUDA\Emgu.CV.Example\XamarinForms\WPF\XamarinForms.WPF.csproj	1
